modulegen/context_test.go

```diff
@@ -95,3 +95,9 @@ func TestModulesHasDependabotEntry(t *testing.T) {
 		assert.True(t, found, "module %s is not present in the dependabot updates", module)
 	}
 }
+
+func getTestRootContext(t *testing.T) *Context {
```

Comment:
We need this rename to avoid collisions, right?

Reply:
Exactly ! It could be avoided if the tests are migrated to a main_test package but this is a bigger work